src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/OffenderSearchServiceTest.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.service
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.BDDMockito.given
+import org.mockito.BDDMockito.then
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import reactor.core.publisher.Mono
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.client.OffenderSearchApiClient
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.offendersearchapi.OffenderSearchByPhraseRequest
+
+@ExtendWith(MockitoExtension::class)
+@ExperimentalCoroutinesApi
+class OffenderSearchServiceTest {
+
+  private lateinit var offenderSearch: OffenderSearchService
+
+  @Mock
+  private lateinit var offenderSearchApiClient: OffenderSearchApiClient
+
+  @BeforeEach
+  fun setup() {
+    offenderSearch = OffenderSearchService(offenderSearchApiClient)
+  }
+
+  @Test
```

Comment:
Can we add a happy path when search returns results

Reply:
Done